play23-core/src/test/scala/play/api/test/ops/AsyncResultExtractorsSpec.scala

```diff
@@ -60,9 +60,9 @@ class AsyncResultExtractorsSpec extends AsyncFreeSpec with AsyncResultExtractors
     }
   }
 
-  def method(name: String) = s"AsyncResultExtrators.$name"
+  def testSubject(methodName: String) = s"play25.AsyncResultExtrators.$methodName"
```

Comment:
should this be
def testSubject(methodName: String) = s"**play23**.AsyncResultExtrators.$methodName"